README.md

```diff
@@ -62,6 +61,11 @@ docker version
 $ brew install git make jq qemu
 ```
 
+#### On Apple Silicon (ARM64) 
+
+The build of QEMU 5.2.0, from Brew has an errata related to Just-In-Time code compilation being broken (running a VM fails with "```Could not allocate dynamic translator buffer```"). Please build a newer release of QEMU (minimal version 6.0.0), from QEMU.org, or wait for this to be fixed in Brew. 
```

Comment:
Can you link to the erratum issue from here? 

Also, more importantly, can you link to the open Homebrew issue for bumping qemu to 6.0.0, so that people can know that once it is resolved, we can just use the version from brew?